app/styles/nypr-ui/_colors.scss

```diff
@@ -26,3 +26,43 @@ $graywhite: #f8f8f8;            // rgb(248, 248, 248)
 // Social
 $facebook: #3b5998;             // rgb(59, 89, 152)
 $facebook-hover: #2a457c;
+
+// Image Filters
+$filter: rgba(0, 0, 0, .3);
```

Comment:
should this have a more specific name? `$filter` doesn't really tell me anything, especially b/c its value is is a very transparent black.